src/vector/feature.rs

```diff
@@ -625,13 +624,13 @@ impl<'a> Iterator for FieldValueIterator<'a> {
             let field_defn =
                 unsafe { gdal_sys::OGR_F_GetFieldDefnRef(self.feature.c_feature, idx as c_int) };
             let field_name = unsafe { gdal_sys::OGR_Fld_GetNameRef(field_defn) };
```

Comment:
`field_name` is owned by `field_defn` so we wont need to free it, right?

Reply:
Yes, it's owned by the field (I think it's a general pattern with `GetFooRef()`).